deploy/kubernetes/helm/openpitrix/values.yaml

```diff
@@ -0,0 +1,156 @@
+namespace: openpitrix-system
+version: v0.3.1
+
+image: openpitrix/openpitrix:v0.3.1
+flyway_image: openpitrix/openpitrix:flyway-v0.3.1
+dashboard_image: openpitrix/dashboard:latest
+watcher_image: hevienz/watcher:latest
```

Comment:
What is this image used for? It will be better if upload the image to openpitrix/watcher

Reply:
This image watches a file changing, if so, he put the content of the new file to etcd.